app_feup/lib/view/Widgets/generic_card.dart

```diff
@@ -94,7 +94,16 @@ class GenericCardState extends State<GenericCard> {
                             padding: EdgeInsets.symmetric(horizontal: 15),
                             margin: EdgeInsets.only(top: 15, bottom: 10),
                           )),
-                          this.getDeleteIcon(context)
+                          Container(
+                            child: this.getMoveIcon(context),
+                            alignment: Alignment.center,
+                            margin: EdgeInsets.only(top: 15, bottom: 10),
+                          ),
+                          Flexible(
+                            child: Container(
+                              child: this.getDeleteIcon(context),
+                              alignment: Alignment.centerRight,
```

Comment:
I know that specific height is not a good practice, but in this case we need to limit the height of the container in order to resolve a little bug. 
When enabling the edit mode everything goes downs some pixels because this container is larger then the others and make the height of the title row grows.
This resolves. The row will still immobilized.

```suggestion
                              alignment: Alignment.centerRight,
                              height: 32
```